src/test/java/com/cloudbees/hudson/plugins/folder/config/AbstractFolderConfigurationTest.java

```diff
@@ -43,10 +47,21 @@ public void testInitialization() {
                 emptyIterable());
     }
 
+    @Test
+    public void healthMetricsAppearsInConfiguredGlobally() throws Exception {
+        HtmlForm cfg = r.createWebClient().goTo("configure").getFormByName("config");
+        assertThat("adding metrics from Global Configuration",
+                AbstractFolderConfiguration.get().getHealthMetrics(), hasSize(cfg.getElementsByAttribute("div", "suffix", "healthMetrics").size()));
+    }
+
     @Issue("JENKINS-60393")
     @Test
     public void shouldBeAbleToRemoveHealthMetricConfiguredGlobally() throws Exception {
-        assertThat("by default, global configuration should have all folder health metrics",
+        List<FolderHealthMetric> healthMetrics = new ArrayList<>(1);
+        healthMetrics.add(new WorstChildHealthMetric(true));
+        AbstractFolderConfiguration.get().setHealthMetrics(healthMetrics);
+
+        assertThat("by default, global configuration should not have any folder health metrics",
```

Comment:
Remove this assertion since **now** it doesn't make sense and it's already checked in `FolderTest`